packages/guides/src/Nodes/Configuration/ConfigurationTab.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\Nodes\Configuration;
+
+use phpDocumentor\Guides\Nodes\CodeNode;
+
+use function hash;
+
+final class ConfigurationTab
+{
+    public readonly string $hash;
+
+    public function __construct(
+        public readonly string $label,
+        public readonly string $slug,
+        public readonly CodeNode $content,
+    ) {
+        $this->hash = hash('sha1', $content->getValue());
```

Comment:
```suggestion
        $this->hash = hash('xxh128', $content->getValue());
```

This is a fast algorithm with less chance of conflict than sha1.